grafana-plugin/src/models/schedule/schedule.ts

```diff
@@ -76,6 +76,9 @@ export class ScheduleStore extends BaseStore {
   @observable
   byDayOptions: SelectOption[];
 
+  @observable
+  scheduleId: string;
```

Comment:
did not get why are you storing scheduleId in schedule store, you can get schedule id in such way
```
 const { query: { id: scheduleId } } = this.props;
```
within  Schedule page

Reply:
`query` is not available on most of the `this.props` except for `Schedules` main page, previously there was some prop drilling on that - `Rotations`, `ScheduleFinal`, `ScheduleOverrides`. Why pass it down to every descendant when you can access it instead from the store?